### ISSUE TITLE
Incorrect flag for valid use of 'not' condition

### ISSUE THREAD

**Issue by grantalewis** [keesschollaart81/vscode-home-assistant#340]
```YAML
  - alias: Trigger Son Asleep When No Alarm Set
    initial_state: 'on'
    trigger:
      - platform: time_pattern
        minutes: '/5'
        seconds: 00
    condition:
      condition: and
      conditions:
        - condition: template
          value_template: >
            {{ is_state('light.son_bedroom_son_bedroom_fan_light','off') and is_state('light.son_bedroom_son_bedroom_fan','on') }}
        - condition: state
          entity_id: input_boolean.son_awake
          state: 'on'
        - condition: not
          conditions:
            - condition: time
              after: '07:00:00'
            - condition: time
              before: '22:00:00'
    action:
      - service: input_boolean.turn_off
        entity_id: input_boolean.son_awake
```
<img width="806" alt="Screen Shot 2020-05-23 at 8 31 46 AM" src="https://user-images.githubusercontent.com/9505044/82731326-fd4dfa80-9cd3-11ea-8d29-1d8dc9008ea5.png">

**Comment by denniswham** [comment]
#337

**Comment by grantalewis** [comment]
FYI applied the update to version: 2.3.0 but am still seeing "Value not accepted.

<img width="398" alt="Screen Shot 2020-05-30 at 8 59 40 PM" src="https://user-images.githubusercontent.com/9505044/83342066-b30edf80-a2b8-11ea-9473-6f79c025d0ac.png">

**Comment by denniswham** [comment]
VSCode Assistant 1.6.5 is still the latest release, and that PR was created after the release, so we have to wait a little longer till a next release.

**Comment by grantalewis** [comment]
Just checking: any progress on this? I'm still seeing the issue.

**Comment by frenck** [comment]
Fixed in 1.7.0
